server/games/typedefs.py

```diff
@@ -154,6 +155,7 @@ def from_basic(
         validity: ValidityState,
         team_outcomes: List[GameOutcome],
         commander_kills: Dict[str, int],
+        team_army_results: List[List[OrderedDict]],
```

Comment:
Can we make this a `List[ArmyResult]` and only convert the inner ones to dictionaries in the `to_dict` method? I think it would work fine if you add a `to_dict` method for `TeamRatingSummary`.

Reply:
Do you mean `List[List[ArmyResult]]`? Each team could have more than one.

Reply:
Yea, my bad